DataLink.tex

```diff
@@ -735,19 +736,22 @@ \subsubsection{VOTable output}
 The name and UCD attributes for FIELD elements in the VOTable
 (and the units in one case) are specified above (see \ref{sec:listOfLinks}).
 
-The DALI specification states that a standardID INFO element with
-name ``standardID'' and the actual standardID string as a value SHOULD
-be provided.  It is recommended to include such an element to help users
-and applications to identify VOTables as results of DataLink services
-this way:
+The DALI specification states that VOTable output should include an
+INFO element with \attval{name}{standardID} and the standardID string as a value.
 \begin{verbatim}
 <RESOURCE type="results">
   ... 
   <INFO name="standardID" value="ivo://ivoa.net/std/DataLink#links-1.0"/> 
   ... 
+  <TABLE>
+  ...
+  </TABLE>
+  ...
 </RESOURCE>
 \end{verbatim}
-
+As of this version, the {links} response {\bf must} include this INFO element
```

Comment:
Suggest replacing "As of this version" with "From version 1.1 of this standard".  Otherwise the text is likely to remain in future revisions and be confusing.

Reply:
Also: DALI 1.1 4.4.3 says "The standardID of a service specification is the IVOA resource identifier for the StandardsRegExt record not including capability-specific fragments."  Does that mean that the value attribute in this example should be `value="ivo://ivoa.net/std/DataLink"` instead of `value="ivo://ivoa.net/std/DataLink#links-1.0"` ?

Reply:
yes, that's true of the standardID of the specification as used in a StandardsRegExt document. Here is is the standardID of the links endpoint itself, which is defined within that spec.

So I think it's OK.